explain/insights/io-heavy.md

```diff
@@ -0,0 +1,17 @@
+---
+explain_insight: 'i/o-heavy'
+title: 'EXPLAIN - Insights: I/O Heavy'
+backlink_href: /docs/explain/insights
+backlink_title: 'Documentation: EXPLAIN - Insights'
+---
+
+**Description:**
+
+Postgres will often spend most of its I/O time on a query in a small set of plan nodes. This insight identifies the most expensive nodes (according to time spent doing I/O) to guide optimization. Since I/O is usually the bottleneck in many database queries, focusing on nodes performing
+the bulk of plan I/O is a good rule of thumb.
```

Comment:
I think “plan I/O” is a bit odd here - maybe just “I/O”

Reply:
Yeah, I think you're right. It read a little funny to me just bare so I added in a filler word, but it does work without it.